transcendence-app/src/auth/auth.controller.ts

```diff
@@ -16,16 +18,17 @@ import { OAuth42Guard } from './oauth42.guard';
 @Controller('auth')
 export class AuthController {
   @Get('login')
-  @ApiFoundResponse()
-  @ApiBadGatewayResponse()
+  @ApiOkResponse({ description: 'Login successfully' })
+  @ApiFoundResponse({ description: 'Redirect to 42 OAuth server' })
+  @ApiBadGatewayResponse({ description: 'Bad Gateway' })
   @UseGuards(OAuth42Guard)
-  async oauth42Login() {
-    return 'Login successfully';
+  oauth42Login() {
+    // Guard implementation
   }
 
-  @Get('logout')
+  @Delete('logout')
```

Comment:
According to the [PassportJS documentation](https://www.passportjs.org/concepts/authentication/logout/), it's better to use POST or DELETE for the logout endpoints.

Reply:
I didn't know it could be done with all of those HTTP methods and I thought, from what I read when I did the webserv, that DELETE method was becoming obsolete, but if Passport recomends it, I have nothing to say about it.